org.gnupg.GPA.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: org.gnupg.GPA
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: gpa
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  #- --filesystem=~/.gnupg:rw
+  #- --filesystem=xdg-run/gnupg:rw
+  - --filesystem=host
+  - --filesystem=/tmp
+
+rename-desktop-file: gpa.desktop
+
+modules:
+
+  - shared-modules/gtk2/gtk2.json
+
+  - name: gpa
+    cleanup:
+      - "*.a"
+      - "*.la"
+      - "/cache"
+      - "/include"
+      - "/lib/*.a"
+      - "/lib/pkgconfig"
+      - "/man"
+      - "/share/aclocal"
+      - "/share/devhelp"
+      - "/share/gtk-doc"
+      - "/share/man"
+      - "/share/pkgconfig"
+    buildsystem: autotools
+    post-install:
+      - install -Dm644 ${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
```

Comment:
you need to edit the .desktop file to set the icon name. Like it was done before.

Reply:
I've applied your previous `rename-icon` suggestion, `test -f build/*/share/app-info/icons/flatpak/128x128/org.gnupg.GPA.png` now passes locally.